samples/bluetooth/mesh/chat/README.rst

```diff
@@ -85,7 +79,21 @@ LEDs:
    Show the OOB authentication value during provisioning if the "Push button" OOB method is used.
 
 Terminal emulator:
-   Used for the interraction with the sample.
+   Used for the interaction with the sample.
+
+Configuration
+*************
+
+|config|
+
+Source file setup
+=================
+
+This sample is split into four source files:
```

Comment:
It says four source files, but there is only mention 3 files. There is only these 3 files. What about just writing "This sample is split into the following source files:" (Same for light and light_ctrl)

Reply:
Changed to "This sample is split into the following source files:" for all samples.